src/libcalamaresui/utils/Paste.cpp

```diff
@@ -40,11 +42,16 @@ logFileContents()
         return QByteArray();
     }
     QFileInfo fi( pasteSourceFile );
-    if ( fi.size() > 16_KiB )
+    if( sizeLimitKiB < 0 )
+        sizeLimitKiB = 1024;
+    qint64 sizeLimitBytes = CalamaresUtils::KiBtoBytes( ( unsigned long long ) sizeLimitKiB );
```

Comment:
Please use `static_cast<>` rather than C-style cast. The `sizeLimitBytes` can be const, as well.